src/mb_sprintf.php

```diff
@@ -29,20 +29,22 @@ public static function mb_vsprintf($format, $argv, $encoding=null) {
       
         // Split the format in two parts: $pre and $post by the first %-directive
         // We get also the matched groups
-        list ($pre, $sign, $filler, $align, $size, $precision, $type, $post) =
+        @list ($pre, $sign, $filler, $align, $size, $precision, $type, $post) =
             preg_split("!\%(\+?)('.|[0 ]|)(-?)([1-9][0-9]*|)(\.[1-9][0-9]*|)([%a-zA-Z])!u",
                        $format, 2, PREG_SPLIT_DELIM_CAPTURE) ;
 
         $newformat .= mb_convert_encoding($pre, $encoding, 'UTF-8');
         
         if ($type == '') {
           // didn't match. do nothing. this is the last iteration.
+          $newargv = array_merge($newargv, array_values($args));
```

Comment:
`$argv` instead of `$args`?

Reply:
Yes sorry I changed some variables in my own repository and forgot to use the old naming in your repository, I'll fix it ^^

Reply:
Done.